src/Speckle.Objects/Geometry/Circle.cs

```diff
@@ -67,9 +51,9 @@ public List<double> ToList()
   {
     var list = new List<double>();
 
-    list.Add(radius ?? 0);
-    list.Add(domain?.start ?? 0);
-    list.Add(domain?.end ?? 1);
+    list.Add(radius);
```

Comment:
this is the only change i'm not 100% sure about, will check some circle curves in breps for deserialization issues

Reply:
so any `null`s in json will become `0` post this change... this is only a problem for the `end` part of the domain, since sometimes, like here, we're defaulting to `1`

so the deserailizer won't throw, but conversions might.
I looked through the latest v2 codebase, and the current v3 codebase, and I can't see anywhere that might potentially leave a null end point... perhaps some historical data might, but I think we'll have to live with some loss v2 -> v3